### PR TITLE
Triage checker

### LINKED ISSUE #56: Ping team lead if triage is unchecked
Pint a team lead if triage checkbox is unchecked and therefore not completed.

Define who the team leads are and define what ping means.

Is this something we want to add to conductor? cc @stdavis @jacobdadams

### PR DESCRIPTION
**Pull request by stdavis**
TODO:
- [x] implement `grade` method
- [x] fix conductor tests that are broken because the issue body is not mocked correctly

Would love to know if this is headed in the right direction...

Closes #56

### CODE REVIEW

**Comment by stdavis** [comment]
OK. This is ready for review. Tell me how it could be better. Here's the output from `test_conductor`:

```
➜ test_conductor
starting conductor...
gathering issues...
collecting Add Geochronology layers from UGS to agol
collecting Add Society.Schools HigherEd to SGID
collecting Remove Society.Schools from SGID
collected 3 issues
punching ticket for triage - 64
No triage task found for team: Data
No triage task found for team: Dev
No triage task found for team: Cadastre
could not find metadata in Add Geochronology layers from UGS to agol
punching ticket for triage - 61
Data Team punched
Dev Team punched
No triage task found for team: Cadastre
punching ticket for Society.Schools_HigherEd
internal sgid punched
sgid 10 punched
meta table punched
stewardship sheet punched
punching ticket for triage - 60
Data Team punched
Dev Team punched
No triage task found for team: Cadastre
punching ticket for Society.Schools
internal sgid punched
sgid 10 punched
meta table punched
open sgid punched
open data punched
arcgis online punched
stewardship sheet punched
all tickets punched
finished grading reports
comment below would be posted to issue: "Add Society.Schools HigherEd to SGID" in production
## conductor results for triage - 61

| check | status |
| - | :-: |
| triage - Data Team | :no_entry: @ZachBeck has not yet performed triage for this issue |
| triage - Dev Team | :no_entry: @steveoh has not yet performed triage for this issue |
comment below would be posted to issue: "Add Society.Schools HigherEd to SGID" in production
## conductor results for Society.Schools_HigherEd

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| sgid10 | :no_entry: |
| meta table |  |
| - item id | :no_entry: |
| - item name | :no_entry: |
| stewardship | :no_entry: Did not find Society.Schools_HigherEd in the worksheet |
comment below would be posted to issue: "Remove Society.Schools from SGID" in production
## conductor results for triage - 60

| check | status |
| - | :-: |
| triage - Data Team | :no_entry: @gregbunce has not yet performed triage for this issue |
| triage - Dev Team | :no_entry: @steveoh has not yet performed triage for this issue |
comment below would be posted to issue: "Remove Society.Schools from SGID" in production
## conductor results for Society.Schools

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| sgid10 | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - deprecation issue link | :no_entry: |
finished
```

Here are the comments that it would post for today's issues. Not sure I like that it creates separate comments for the triage tasks...

comment below would be posted to issue: "Add Society.Schools HigherEd to SGID" in production
## conductor results for triage - 61

| check | status |
| - | :-: |
| triage - Data Team | :no_entry: @ZachBeck has not yet performed triage for this issue |
| triage - Dev Team | :no_entry: @steveoh has not yet performed triage for this issue |
comment below would be posted to issue: "Add Society.Schools HigherEd to SGID" in production
## conductor results for Society.Schools_HigherEd

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| sgid10 | :no_entry: |
| meta table |  |
| - item id | :no_entry: |
| - item name | :no_entry: |
| stewardship | :no_entry: Did not find Society.Schools_HigherEd in the worksheet |
comment below would be posted to issue: "Remove Society.Schools from SGID" in production
## conductor results for triage - 60

| check | status |
| - | :-: |
| triage - Data Team | :no_entry: @gregbunce has not yet performed triage for this issue |
| triage - Dev Team | :no_entry: @steveoh has not yet performed triage for this issue |
comment below would be posted to issue: "Remove Society.Schools from SGID" in production
## conductor results for Society.Schools

| check | status |
| - | :-: |
| internal sgid | :no_entry: |
| sgid10 | :no_entry: |
| meta table | :no_entry: |
| open sgid | :no_entry: |
| open data | :no_entry: |
| arcgis online | :no_entry: |
| stewardship |  |
| - deprecation issue link | :no_entry: |
finished

File: tests/data/triage_issue_body_strikethrough.txt [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+<!--
+When the champion from your team has completed the triage, check [x] the checkbox
+-->
+
+# Triage
+
+- [ ] Data Team Triage (@gregbunce)
+- [ ] Dev Team Triage (@steveoh)
+~- [ ] Cadastre Team Triage (@rkelson)~
```
Comment (steveoh) [inline tests/data/triage_issue_body_strikethrough.txt:22]: I think putting the `~` on the right side of the `-` keeps the formatting as a list. We might want to test for both.

**Review by steveoh — APPROVED** [review]
looks good. Do you want to add the triage comments to the existing table or what changes can be made to improve your dislike of the multiple comments?

**Comment by stdavis** [comment]
I just didn't like the idea of everyone getting two email instead of one but I wasn't sure how to group it as part of the other tables since they are based on table names. Seems like it would take a more fundamental change.

**Comment by steveoh** [comment]
I was thinking we should start to edit the original bot comment instead of leaving multiple. Or hide or delete the old ones.

**Comment by stdavis** [comment]
Would people get notifications if we edited the original comment? I kind of like the idea of having a history of how many times it has run and what has happened (or not happened) in between runs. But it's not a big deal to me either way.

**Comment by steveoh** [comment]
Probably not. We could try to roll up the priors into a new one?

**Comment by stdavis** [comment]
I'm not following. Can you give an example?

**Comment by steveoh** [comment]
Sure. I wonder if the bot can delete it's prior comment and then create a new one like

## Issue open for N days/weeks

### Current results

- stuff
- more stuff

### Triage stuff

- steve is being lazy

### N Prior runs

<details>
<summary>x/x/x results</summary>
- stuff
- more stuff
</details>

<details>
<summary>x/x/x results</summary>
- stuff
- more stuff
</details>

<details>
<summary>x/x/x results</summary>
- stuff
- more stuff
</details>

**Comment by stdavis** [comment]
I think that could work. I think that I like posting a new comment each time slightly better. It would be easier to see how they fit into the history of the issue. And then only folks involved in the current triage would be notified rather than re-pinging people with the data under "N Prior runs".

**Comment by steveoh** [comment]
What about if we sent the triagers an email directly instead of leaving a github comment. conductor is creating the notifications already. maybe they need more direct nagging?

**Pull request merged**